src/Weeder.hs

```diff
@@ -333,21 +331,14 @@ typeToNames (Roll t) = case t of
 
 analyseExport :: MonadState Analysis m => Module -> AvailInfo -> m ()
```

Comment:
`analyseExport` is actually semi-dead code at the moment (yes, ironic) - `ModuleRoot` is never constructed, so the `exports` field of `Analysis` is never read.

It would be safe to just remove `analyseExport` entirely for now, which would keep compatibility with earlier GHC versions.

Reply:
Isn't it used by `analyseHieFile`?

https://github.com/ocharles/weeder/blob/a959f2e35ac30fff6e9c330e319832a7266e1a5c/src/Weeder.hs#L286

Reply:
Yes, but removing the call to it would not change any behaviour.